.circleci/check_version_trigger_release.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+set -e
+
+#!/usr/bin/env bash
+LATEST_COMMIT=$(git rev-parse HEAD)
+VERSION_COMMIT=$(git log -1 --format=format:%H VERSION)
+if [ $VERSION_COMMIT = $LATEST_COMMIT ];
+    then
+        VERSION=`cat VERSION`
```

Comment:
if VERSION is empty/null/not set abort

Reply:
Implemented where VERSION is empty. it will error if there is no VERSION file from line 6.